src/python/pants/backend/java/dependency_inference/java_parser_test.py

```diff
@@ -77,8 +74,9 @@ def test_simple_java_parser_analysis(rule_runner: RuleRunner) -> None:
                     ],
                 )
 
-                java_sources(
+                java_source(
```

Comment:
This is fine, but also it would have been fine to keep using `java_sources` in the BUILD file too (so long as the RuleRunner registers the relevant rules).

No need to change, only an FYI